articles/active-directory/develop/quickstarts/active-directory-javascriptspa.md

```diff
@@ -74,12 +81,20 @@ var msalconfig = {
 };
 ```
 > [!NOTE]
-> If you use [Visual Studio 2017](https://visualstudio.microsoft.com/downloads/), the Redirect URL will be set to to *http:/<span/>/localhost:44316/* as it is configured in the code sample's project. If you use [Node.js](https://nodejs.org/en/download/) or any other web server, set the redirect URL to `http://localhost:44316/` and then configure the server to start listening this port. 
+> If you use [Visual Studio 2017](https://visualstudio.microsoft.com/downloads/), the redirect URI will be set to to *http:/<span/>/localhost:30662/* as it is configured in the code sample's project. If you use [Node.js](https://nodejs.org/en/download/) or any other web server, set the redirect URI to *http:/<span/>/localhost:30662/* and then configure the server to start listening on this port.
 >
-> For Node.js, you can run the following from the project's directory to start listening on port 44316:
-> ```batch
-> node server.js
-> ```
+
+#### Step 4: Run the project
+
+If using Visual Studio, press **F5** to run your project.
+
+If using Node.js, on a command line you can run the following from the project's directory to start the server:
+ ```batch
+ npm install
+ node server.js
+ ```
+In a browser window navigate to the `http://localhost:30662/` location and you see the **Call Microsoft Graph API** button to start sign-in.
```

Comment:
Suggestion: 
Open the web browser and navigate to `http://localhost:30662/`. Click **Call Microsoft Graph API** button to start sign-in.